### PR TITLE
update next webinar card, move last webinar to correct tab

### PR DESCRIPTION
**Pull request by texas-kristen**

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit 151cbada92b5fefe6d8d144af88ac95401b346f0

https://deploy-preview-1641--stoplightio.netlify.app

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit 41c2662b10ad8f0be486ee70d1e499181f40dc03

https://deploy-preview-1641--stoplightio.netlify.app

File: netlify/case-studies/platform-101-workshop.md [hunk 1]
```diff
@@ -5,19 +5,19 @@ tags:
 relatedTags:
   - upcoming
 publishedDate: 2020-10-10T15:31:24.486Z
-title: "Workshop: Platform 101"
-subtitle: November 3, 2020, 05:00 PM CT
+title: "Workshop: Stoplight Platform 101"
+subtitle: Experts show you leverage the full power of the Stoplight Platform. November 3 @ 09:00 AM CT
```
Comment (paulatulis) [inline netlify/case-studies/platform-101-workshop.md:9]:
```suggestion
subtitle: Experts show you how to leverage the full power of the Stoplight Platform. November 3 @ 09:00 AM CT 
```

File: netlify/settings.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 banners:
   - ends: 1609430400000
     markdown: >-
-      Gain a better understanding of Spectral and Style Guides. Register for the interactive workshop <u
-      class="ml-1">[here](https://us02web.zoom.us/webinar/register/WN_Ne8wUjBVSVqAHkXHgF1Pvg?utm_source=stoplightio&utm_medium=banner&utm_campaign=workshop)</u>.
+      Experts show you leverage the full power of the Stoplight Platform. Register for the interactive workshop <u
```
Comment (paulatulis) [inline netlify/settings.yaml:4]:
```suggestion
      Experts show you how to leverage the full power of the Stoplight Platform. Register for the interactive workshop <u
```

File: netlify/case-studies/platform-101-workshop.md [hunk 3]
```diff
@@ -39,15 +39,10 @@ meta:
     username: '@stoplightio'
 ---
 
-<!-- ## [Register](https://us02web.zoom.us/webinar/register/WN_Ne8wUjBVSVqAHkXHgF1Pvg) -->
+## [Registration](https://us02web.zoom.us/webinar/register/7416038130475/WN_uqv4RV48TguYNArge1sITw)
 
-<!-- Update with registration link once it's made -->
+Join us for an overview of the Stoplight product. Our Solution Consultant will provide a 30-minute demonstration of the core features and functionality to help you see how to get started. Learn how an API design-first methodology can be easy for everyone, not just technical experts. After the demonstration, attendees will be able to ask questions for their specific needs.
```
Comment (paulatulis) [inline netlify/case-studies/platform-101-workshop.md:44]:
```suggestion
Join us for an overview of the Stoplight product. Our Solutions Consultant will provide a 30-minute demonstration of the core features and functionality to help you see how to get started. Learn how an API design-first methodology can be easy for everyone, not just technical experts. After the demonstration, attendees will be able to ask questions for their specific needs.
```

**Review by paulatulis — CHANGES_REQUESTED** [review]

**Review by paulatulis — APPROVED** [review]

**Pull request merged**